frontend/src/screens/learner.tsx

```diff
@@ -21,23 +21,51 @@ const HighlightedStudies: FC = () => {
     const { highlightedStudies } = useParticipantStudies()
     const isMobile = useIsMobileDevice()
 
+    const handleClick = () => {
+        const element = document.getElementById('all-studies-unique-id')
+        element?.scrollIntoView({ behavior: 'smooth' })
+    }
+
     if (!highlightedStudies.length) return null
 
     return (
         <Box bg={colors.navy} py='md'>
             <Container>
-                <Stack>
-                    <Title c='white' order={2}>Highlighted Studies</Title>
-                    {isMobile ?
-                        <MobileStudyCards studies={highlightedStudies} /> :
-                        <DesktopStudyCards studies={highlightedStudies} />
-                    }
-                </Stack>
+                <Flex direction='column'>
```

Comment:
this whole section can be Flex -> Group/Stack

Reply:
Modified